src/components/DropDownButton/index.tsx

```diff
@@ -230,25 +265,34 @@ class BaseDropDownButton extends React.Component<
 						dropUp={dropUp}
 						alignRight={alignRight}
 						onClick={this.toggle}
-						minWidth={`${this.state.minWidth}px`}
-						maxHeight={px(this.props.listMaxHeight)}
+						maxHeight={px(this.props.listMaxHeight ?? 300)}
 					>
-						{React.Children.map(children, (child, i) => {
-							if (noListFormat) {
-								return child;
-							}
-							if (!child) {
-								return;
-							}
-							if ((child as any).type === Divider) {
-								return child;
-							}
-							return (
-								<Item border={Boolean(border && i)} key={i}>
-									{child}
-								</Item>
-							);
-						})}
```

Comment:
I'm not too sure about removing the Child concept... I would keep both, as the Select component does.

Reply:
The child concept is not entirely removed, it has just been moved to `item.content` @JSReds